src/main/kotlin/me/zeroeightsix/kami/gui/KamiImgui.kt

```diff
@@ -95,6 +95,10 @@ object KamiImgui {
             }
             else -> 110
         }
+
+        ImGui.getIO().addConfigFlags(1 shl 6) // 1<<6 should be the DockingEnable flag
```

Comment:
It is actually public https://github.com/SpaiR/imgui-java/blob/master/example/src/main/java/Main.java#L27

Reply:
That's bizarre. IntelliJ didn't seem to find it.